worlds/aquaria/Regions.py

```diff
@@ -0,0 +1,1377 @@
+        add_rule(self.world.get_location("Open water bottom left area, bulb inside the downest fish pass", self.player),
+                 lambda state: _has_fish_form(state, self.player))
+        add_rule(self.world.get_location("Kelp forest bottom left area, Walker baby", self.player),
+                 lambda state: _has_spirit_form(state, self.player))
+        add_rule(self.world.get_location("The veil top left area, bulb hidden behind the blocking rock", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Turtle cave, Turtle Egg", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Abyss left area, bulb in the bottom fish pass", self.player),
+                 lambda state: _has_fish_form(state, self.player))
+        add_rule(self.world.get_location("Song cave, Anemone seed", self.player),
+                 lambda state: _has_nature_form(state, self.player))
+        add_rule(self.world.get_location("Song cave, Verse egg", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Verse cave right area, Big Seed", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Arnassi ruins, Song plant spore on the top of the ruins", self.player),
+                 lambda state: _has_beast_form(state, self.player))
+        add_rule(self.world.get_location("Energy temple first area, bulb in the bottom room blocked by a rock",
+                                         self.player), lambda state: _has_energy_form(state, self.player))
+        add_rule(self.world.get_location("Home water, bulb in the bottom left room", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Home water, bulb in the path bellow Nautilus Prime", self.player),
+                 lambda state: _has_bind_song(state, self.player))
+        add_rule(self.world.get_location("Naija's home, bulb after the energy door", self.player),
+                 lambda state: _has_energy_form(state, self.player))
+        add_rule(self.world.get_location("Abyss right area, bulb behind the rock in the whale room", self.player),
+                 lambda state: _has_spirit_form(state, self.player) and
+                               _has_sun_form(state, self.player))
+        add_rule(self.world.get_location("Arnassi ruins, Arnassi Armor", self.player),
+                 lambda state: _has_fish_form(state, self.player) and
+                               _has_spirit_form(state, self.player))
+
+
+
+
+    def __excluded_hard_or_hidden_location(self) -> None:
+        self.world.get_location("Energy temple boss area, Fallen god tooth", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
```

Comment:
```suggestion
        self.world.get_location("Energy temple boss area, Fallen god tooth", self.player).progress_type = 
            LocationProgressType.EXCLUDED
```

I don't think you need parentheses for any of these (and some of these don't have them already)

Reply:
Hmmm. Not sure the code you propose work actually. In Python, we must use \ at the end of a line if the instruction si split in multiple line. Unless the new line is inside bracket or parenthesis or other delimiter. But I will put them more coherent.

Reply:
Whichever you prefer -- I'm in the office right now so I can't really check whether these suggestions work as-is, unfortunately

Reply:
I have modified them. I will push after testing the bool(...) that we talk in another comment.